packages/sanity/playwright-ct.config.ts

```diff
@@ -92,6 +92,7 @@ export default defineConfig({
           firefoxUserPrefs: {
             'dom.events.asyncClipboard.readText': true,
             'dom.events.testing.asyncClipboard': true,
+            'ui.prefersReducedMotion': 1,
```

Comment:
why was this added? (would be great to have a comment)

Reply:
The idea is to speed up the tests by reducing the CSS animations (less CPU time used).